core/blockly.js

```diff
@@ -107,13 +108,11 @@ Blockly.EventData;
 /**
  * Returns the dimensions of the specified SVG image.
  * @param {!SVGElement} svg SVG image.
- * @return {!Object} Contains width and height properties.
+ * @return {!Blockly.utils.Size} Contains width and height properties.
  */
 Blockly.svgSize = function(svg) {
-  return {
-    width: svg.cachedWidth_,
-    height: svg.cachedHeight_
-  };
+  svg = /** @type {?} */ (svg);
```

Comment:
Because an SVG element doesn't have `cachedHeight_` and `cachedHeight_`, these are properties we've added onto the element. It is for this reason that I'm not a fan of adding properties onto DOM elements.